lib/cs/README.md

```diff
@@ -0,0 +1,5 @@
+Instead of reinventing the wheel of Nuget, we want to utilize the power
```

Comment:
Can you reword this `reinventing the wheel` 😅 
"Utilizing nuget's own capabilities"